core/domain/html_validation_service.py

```diff
@@ -852,6 +855,118 @@ def add_dimensions_to_image_tags(exp_id, html_string):
     return python_utils.UNICODE(soup).replace('<br/>', '<br>')
 
 
+def generate_math_svgs_filename(height, width, vertical_padding):
+    """Generates a random filename for a math SVG based on the given
+    dimensions.
+
+    Args:
+        height: str. Height of the image.
```

Comment:
Units? (Here and elsewhere.)

Reply:
this is not actual height, but a string from which we decode the actual height.
We need this because we are planning to use the relative units of ex as a dimensions for SVGs.
For example an Svg has height and width as 1.343ex and 4.324ex , the encoded height and width would be `1d343` and `4d324` (We have replaced the decimal point with a letter because we can't use dot in the name). 
Anyway I've added updated the docstrings